network/README.md

```diff
@@ -4,6 +4,7 @@ _Lists the geographical and physical characteristics of collection points._
 
 ### FILES ###
 
+* `features.csv` - Physical site location descriptions.
```

Comment:
I think this one needs to be removed from here, as the `features.csv` is now under `/environment `?

Reply:
agree